R/util.R

```diff
@@ -57,6 +57,27 @@ get_vaccine_option_description <- function(vaccine_option) {
   )
 }
 
+get_pathogen_description <-  function(pathogen_id) {
+  # get pathogen information from the package and generate
+  # description including R0 and IFR range (across all countries)
+  # for the pathogen
+
+  infection <- daedalus::daedalus_infection(pathogen_id)
+  country_ifrs <- sapply(daedalus::country_names, function(country_name) {
+    country <- daedalus::daedalus_country(country_name)
+    weighted.mean(
+      infection$ifr, country$demography
+    )
+  })
```

Comment:
So a more lightweight version can use the raw `country_data` provided with the package and bypass `daedalus_country()`. Not strong feelings though.

```r
z = daedalus::infection("sars_cov_1")

all_ifr_values = vapply(daedalus::country_data, function(le){
    weighted.mean(z$ifr, le$demography)
}, 1.0)

# get the min and max
ifr_range = range(all_ifr_values)
```